docs/diagrams/StorageClassDiagram.puml

```diff
@@ -19,6 +19,9 @@ Class "<<interface>>\nEduLogStorage" as EduLogStorage
 Class JsonEduLogStorage
 Class JsonSerializableEduLog
 Class JsonAdaptedStudent
+Class JsonAdaptedLesson
+Class JsonAdaptedDay
+Class JsonAdaptedTime
```

Comment:
This class is yet to be implemented right?

Reply:
yep, not implemented yet, but will be pursued as part of #113